### ISSUE TITLE
How would I check for the latest version?

### ISSUE THREAD

**Issue by luebken** [upbound/up#235]
### What problem are you facing?
The `up` CLI has an ever-growing awesome list of capabilities which we release on a rapid pace. As a user, I'm wondering if I'm using the latest version. 

### How could Upbound help solve your problem?

I understand that I can check https://github.com/upbound/up/releases but that seems a bit hidden. I'm wondering if we can make that somewhat more obvious. Also I'm wondering if this could be implemented into the CLI but I haven't found a good example for this.

Epic #

**Comment by hasheddan** [comment]
@luebken I've seen some CLI's check if they are up to date when someone runs `up --version` and if not, gives them a prompt to upgrade to latest. Alternatively, we could opt to add an `up upgrade` / `up upgrade --check` command that downloads a new version for you / tells you if there is a new version to upgrade to, but I am less excited about that given that folks may be managing their `up` installation via a variety of package managers.

**Comment by luebken** [comment]
@hasheddan Agree. Also an implicit "phone home" functionality might not always be what security aware users might want.
